client/src/store/modules/scheduledEvent/actions.js

```diff
@@ -113,5 +114,31 @@ export default {
                 root: true
             });
         }
+    },
+
+    [actions.UPDATE_EVENT]: async ({ commit, dispatch }, event) => {
```

Comment:
In this task, action should not be used to update all fields of the event - you need to update one field - the status.

Reply:
In this case we'll need to make a lot action update event, update start date, etc. In this way we could use just one and pass parameter to it.